### PR TITLE
test(12h with asymmetric cluster): Add new longevity-50gb-asymmetric-cluster-12h test

### PR DESCRIPTION
**Pull request by juliayakovlev**
This test based on longevity-200gb-48h. New test will run 12h on cluster where
Scylla starts with random SMP on every node

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: jenkins-pipelines/longevity-50gb-asymmetric-cluster-12h.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-50GB-12h-verifier-LimitedMonkey-tls.yaml", "configurations/db-nodes-shards-random.yaml"]''',
+
+    timeout: [time: 780, unit: 'MINUTES']
```
Comment (bentsi) [inline jenkins-pipelines/longevity-50gb-asymmetric-cluster-12h.jenkinsfile:12]: remove

Comment (bentsi) [reply jenkins-pipelines/longevity-50gb-asymmetric-cluster-12h.jenkinsfile:12]: we use test duration to dynamically calculate the sct timeout

Comment (juliayakovlev) [reply jenkins-pipelines/longevity-50gb-asymmetric-cluster-12h.jenkinsfile]: done

File: test-cases/longevity/longevity-50GB-12h-verifier-LimitedMonkey-tls.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+nemesis_interval: 30
+nemesis_during_prepare: false
+
+user_prefix: 'longevity-50gb-12h'
+
+space_node_threshold: 644245094
+
+use_mgmt: true
+
+#server_encrypt: true
```
Comment (bentsi) [inline test-cases/longevity/longevity-50GB-12h-verifier-LimitedMonkey-tls.yaml:25]: uncomment

Comment (juliayakovlev) [reply test-cases/longevity/longevity-50GB-12h-verifier-LimitedMonkey-tls.yaml]: done

**Review by bentsi — APPROVED** [review]

**Comment by juliayakovlev** [comment]
Need be backported to 2022.1 and 2021.1 - https://github.com/scylladb/qa-tasks/issues/840#issuecomment-1359508301

**Pull request merged**